react-bootstrap/index.d.ts

```diff
@@ -590,6 +590,7 @@ declare namespace ReactBootstrap {
         active?: boolean;
         noCaret?: boolean;
         eventKey?: any;
+        title?: any; // TODO: Add more specific type
     }
     type NavDropdownProps = NavDropdownBaseProps & React.HTMLProps<NavDropdown>;
```

Comment:
I'm totally stumped by this, my unfamiliarity with react isn't helping. @DanielRosenwasser would you know what to do here?